### PR TITLE
activate dynamic and closable param in Tabs documentation #1387

### PR DESCRIPTION
**Pull request by halilbay**

### CODE REVIEW

File: examples/reference/layouts/Tabs.ipynb [hunk 1]
```diff
@@ -180,6 +183,13 @@
    "source": [
     "pn.Row(tabs, tabs.clone(active=1, tabs_location='right'), tabs.clone(active=2, tabs_location='below'), tabs.clone(tabs_location='left'))"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (philippjfr) [inline examples/reference/layouts/Tabs.ipynb:192]: Please delete this empty cell.

File: examples/reference/layouts/Tabs.ipynb [hunk 2]
```diff
@@ -113,7 +113,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
```
Comment (philippjfr) [inline examples/reference/layouts/Tabs.ipynb:118]: Please delete this metadata.

File: examples/reference/layouts/Tabs.ipynb [hunk 3]
```diff
@@ -135,7 +137,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "tabs = pn.Tabs(p1, p2, p3)\n",
+    "tabs = pn.Tabs(p1, p2, p3, dynamic=True)\n",
```
Comment (philippjfr) [inline examples/reference/layouts/Tabs.ipynb:140]: I'm -1 on changing this here, both `dynamic=True` and `closable=True/False` need a little section that explains what they do.

**Review by philippjfr — CHANGES_REQUESTED** [review]
Hi @halilbay, welcome and thanks so much for contributing! Rather than just peppering these options into the existing examples we really need little standalone examples that demonstrate what these options actually do. Sorry I didn't explain that in the original issue.

File: examples/reference/layouts/Tabs.ipynb (hunk 3 above)
Comment (halilbay) [reply examples/reference/layouts/Tabs.ipynb:140]: Sorry Philipp, I don't get it. You mean using these both parameters in there like;
'"tabs = pn.Tabs(p1, p2, p3, dynamic=True, closable=True)\n", '?

Comment (philippjfr) [reply examples/reference/layouts/Tabs.ipynb:140]: No, I mean adding a completely new section for each of these and describing in detail what they do.

**Comment by philippjfr** [comment]
@halilbay Let me know if you're still interested in working on this. I realize it's a bit more work than you expected but the whole point is to demonstrate what these options actually do rather than just sticking them in there somewhere.

**Comment by halilbay** [comment]
@philippjfr sorry for latency. I am a little bit busy but I will work on that

**Comment by philippjfr** [comment]
Closing for now, let me know if you need assistance to resubmit a PR.

**Pull request closed without merging**